src/backy/sources/ceph/rbd.py

```diff
@@ -113,9 +113,7 @@ def snap_rm(self, image):
 
     @contextlib.contextmanager
     def export_diff(self, new, old):
-        self.log.info(
-            "export-diff", ceph_version=str(backy.sources.ceph.CEPH_VERSION)
-        )
+        self.log.info("export-diff")
```

Comment:
In the end, we need to decide whether we want to see some hint regarding the utilised Ceph version in the logs or not. When the previous change had been made, we probably wanted to see it for some reason. So in this case we could log the whole output wtring of `ceph -v` as well.

But I am fine with removing logging this information as well.